src/node.cc

```diff
@@ -3061,6 +3071,8 @@ static void PrintHelp() {
          "function is used\n"
          "  --trace-deprecation  show stack traces on deprecations\n"
          "  --v8-options         print v8 command line options\n"
+         "  --warn-on-late-sync  print warning message when sync\n"
+         "                       function is used\n"
```

Comment:
well it's more like "print a warning on usage of most sync after first tick, excluding the `process` `'exit'` event." but how much do we really need to put here?